vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -115,6 +115,98 @@ disableTimesyncd() {
     systemctl stop systemd-timesyncd || exit 1
     systemctl disable systemd-timesyncd || exit 1
     systemctl mask systemd-timesyncd || exit 1
+
+    # Update chrony configuration
+    cat > /etc/chrony/chrony.conf <<EOF
```

Comment:
this file is also created in `disableNtpAndTimesyncdInstallChrony` 

shoudl we simply update the lines we want to update instead of doing a full redrop ?